dev-docs/bidders/equativ.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: bidder
+title: Equativ
+description: Prebid Equativ Bidder Adapter
+biddercode: equativ
+media_types: banner
+tcfeu_supported: true
+coppa_supported: true
+gvl_id: 45
+gpp_supported: true
+schain_supported: true
+usp_supported: true
+userIds: all
+pbjs: true
+pbs: true
+floors_supported: true
+sidebarType: 1
+---
+
+### Registration
+
+The Equativ bidder adapter requires setup and approval from the Equativ service team. Please reach out to your account manager for more information to start using it.
+
+### Bid params
+
+{: .table .table-bordered .table-striped }
+
+| Name | Scope | Description | Example | Type |
+|------|-------|-------------|---------|------|
+| `networkId` | required | The network identifier you have been provided with. _See **Bid Parameter Usage** notes below for more information_. | `1234` | `integer` |
+| `siteId` | required | The placement site ID. _See **Bid Parameter Usage** notes below for more information_. |`1234` | `integer` |
+| `pageId` | required | The placement page ID. _See **Bid Parameter Usage** notes below for more information_. | `1234` | `integer` |
+| `formatId` | required | The placement format ID. _See **Bid Parameter Usage** notes below for more information_. | `1234` | `integer` |
+
+#### Bid Parameter Usage 
+Different combinations of parameters are required depending upon which ones you choose to use.
+
+There are three options for passing bidder parameters:
+
+- **Option 1**.  Specify `networkId` by itself (_without_ `siteId`, `pageId` and `formatId`), or
+- **Option 2**.  Specify `siteId` _and_ `pageId` _and_ `formatId` (all together) _without_ `networkId`, or
+- **Option 3**.  Specify _none_ of the above parameters, and instead use either  `ortb2.site.publisher.id` or `ortb2.app.publisher.id`
+
+See **Sample Banner Setup** for examples of these parameter options.
+
+### Supported Media Types
+
+{: .table .table-bordered .table-striped }
+
```

Comment:
please remove extra space

Reply:
By extra space I'm assuming you mean the line between the table definition and the content.  Marking as resolved.